examples/complete/main.tf

```diff
@@ -234,21 +242,39 @@ locals {
     }
   }
 
-  mission_app_self_mg_node_group = {
-    uds-core_ng = {
+  uds_core_self_mg_node_group = {
+    uds_core_ng = {
       ami_type      = "BOTTLEROCKET_x86_64"
       ami_id        = data.aws_ami.eks_default_bottlerocket.id
-      instance_type = "m5d.2xlarge" # conflicts with instance_requirements settings
+      instance_type = null # conflicts with instance_requirements settings
       min_size      = 3
       max_size      = 5
       desired_size  = 3
-      key_name      = module.key_pair.key_pair_name
+      key_name      = module.self_managed_node_group_keypair.key_pair_name
+
+      block_device_mappings = {
+        xvda = {
+          device_name = "/dev/xvda"
+          ebs = {
+            volume_size = 100
+            volume_type = "gp3"
+          }
+        }
+        xvdb = {
+          device_name = "/dev/xvdb"
+          ebs = {
+            volume_size = 100
+            volume_type = "gp3"
+            #need to add and create EBS key
+          }
+        }
+      }
 
       bootstrap_extra_args = <<-EOT
         # The admin host container provides SSH access and runs with "superpowers".
-        # It is disabled by default, but can be disabled explicitly.
+        # It is disabled by default, enabled here for easy SSH access into bottlerocket nodes with the keypair created by the module.
         [settings.host-containers.admin]
-        enabled = false
+        enabled = true
```

Comment:
is this to serve as a backup to SSM? Does the corresponding security group only allow access from the bastion IP?

Reply:
Really just for the example to make getting into the node easier - this makes SSHing into the bottlerocket nodes easier to get logs out of them via logdog or whatever

Reply:
https://github.com/defenseunicorns/narwhal-delivery-iac-swf-reference-deployment/blob/main/docs/bottlerocket-node-debugging.md

PoC here